plant/plants/tasks.py

```diff
@@ -0,0 +1,11 @@
+from celery import shared_task
+
+from plant.plants.models import PlantFamily
+from plant.plants.requests import fetch_plant_families
+
+
+@shared_task
+def get_plant_families():
+    families = fetch_plant_families()
+    families_list = [PlantFamily(name=val) for val in families]
+    PlantFamily.objects.bulk_create(families_list)
```

Comment:
Problem with bulk_create is that you create duplicated Families with each task execution. I would probably suggest iterating over families and using get_or_create